parser-typechecker/src/Unison/Builtin.hs

```diff
@@ -247,7 +247,6 @@ builtinsSrc =
   , B "Int.signum" $ int --> int
   , B "Int.leadingZeros" $ int --> nat
   , B "Int.negate" $ int --> int
-  , B "Int.negate" $ int --> int
```

Comment:
For some reason there were 2 copies of `Int.negate` in this list. Which has no effect.